.travis.yml

```diff
@@ -57,6 +59,10 @@ after_success:
       docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
       docker push $DOCKER_IMAGE:$DOCKER_TAG;
     fi
+  - if [ "$DOCKER_TAG" == "master" ]; then
```

Comment:
Why is this necessary? If I recall it pushes on any branch that has a tag? And master should only get pushed when it is an actual release AKA has a git tag.